src/core/public/core_app/status/lib/load_status.ts

```diff
@@ -12,11 +12,16 @@ import type { StatusResponse, ServiceStatus, ServiceStatusLevel } from '../../..
 import type { HttpSetup } from '../../../http';
 import type { NotificationsSetup } from '../../../notifications';
 import type { DataType } from '../lib';
-
+export interface MetricMeta {
+  description: string;
+  value?: any | any[];
```

Comment:
The `value` type is either a string or a number or an array of either. I'm open to suggestions what to replace the `any` with here.